packages/@orbit/jsonapi/src/lib/transform-requests.ts

```diff
@@ -33,11 +33,18 @@ export const TransformRequestProcessors = {
       .then((raw: JSONAPIDocument) => {
         let responseDoc: DeserializedDocument = serializer.deserializeDocument(raw);
         let updatedRecord: Record = <Record>responseDoc.data;
-
+        let transforms = [];
         let updateOps = recordDiffs(record, updatedRecord);
         if (updateOps.length > 0) {
-          return [buildTransform(updateOps)];
+          return transforms.push(buildTransform(updateOps));
+        }
+        if (responseDoc.included && responseDoc.included.length > 0) {
+          let includedOpts = responseDoc.included.map(record => {
```

Comment:
Can I just use replace record or I also need to create one operation to replace it and then operations to update every attribute and the remote id?

Reply:
@cibernox one `replaceRecord` operation per record should be sufficient